### PR TITLE
Added a notes file

### PR DESCRIPTION
**Pull request by NinjaEagle**

### CODE REVIEW

File: Notes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Goal: Let landlords create reviews for their tenants and be able to track how good/bad these tenants are so other landlords can decide whether or not to lease the places to them
```
Comment (ashleysurita) [inline Notes.md]: at the end of this sentence, it should say "lease their places"

File: Notes.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Landlords
+    -The primary user model
+    -2 separate show pages ( 1 for the profile page, 1 for the list of tenants)
```
Comment (ashleysurita) [inline Notes.md]: the 2nd show page is supposed to be a tenants show page, index page is different for the list of tenants

File: Notes.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    -Edit their profile info
+    -Sign up page, login page
+    -The site will have a Navbar (top or side) allowing the landlord to navigate to the home page and the profile
+    -Landlord show page- root route (home page?) which includes a Tenants list show page. It shows the list of tenants that have been in contact with landlord
```
Comment (ashleysurita) [inline Notes.md]: this isn't a show page, this is an index page or root route

**Review by ashleysurita — CHANGES_REQUESTED** [review]
Please review and resubmit, let me know if you need clarification.

File: Notes.md (hunk 2 above)
Comment (NinjaEagle) [reply Notes.md]: Gotcha well can you correct it? I can't comment on the PR.

**Review by NinjaEagle — COMMENTED** [review]
Ok. Cool I'll see if I can make another PR.

**Review by NinjaEagle — COMMENTED** [review]
I have included the changes into the notes file.

**Review by ashleysurita — APPROVED** [review]
Looks good!

**Comment by ashleysurita** [comment]
Looks good!

**Pull request merged**